Automattic/newspack-newsletters
# Add new layouts

### All Submissions:






### Changes proposed in this Pull Request:

It's an update to #709.

Basically, this PR adds 2 new layouts: "Center" and "Topper with 2 articles" and uses the new site title and site logo blocks for all the layouts

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Create a new Newsletter
3. Insert the new layouts and send the newsletter
4. Change your header background color
5. Check if the 2 new layouts are applying the same header background color as the site
6. Change your date format
7. If you are using Mailchimp, do you see the merge tag for the date

### Other information:
